.lighthouserc.json

```diff
@@ -0,0 +1,197 @@
+            }
+          },
+          {
+            "matchingUrlPattern": "http://[^/]+/privacy$",
+            "assertions": {
+              "offscreen-images": ["error", {"minScore": 1}],
+              "uses-webp-images": ["error", {"minScore": 1}],
+              "render-blocking-resources": ["error", {"minScore": 0}],
+              "unminified-css": ["error", {"minScore": 0}],
+              "unminified-javascript": ["error", {"minScore": 0}],
+              "unused-css-rules": ["error", {"minScore": 0}],
+              "uses-text-compression": ["error", {"minScore": 0}],
+              "uses-http2": ["error", {"minScore": 0}],
+              "errors-in-console": ["error", {"minScore": 0}]
+            }
+          },
+          {
+            "matchingUrlPattern": "http://[^/]+/teach$",
+            "assertions": {
+              "offscreen-images": ["error", {"minScore": 0}],
+              "uses-webp-images": ["error", {"minScore": 1}],
+              "render-blocking-resources": ["error", {"minScore": 0}],
+              "unminified-css": ["error", {"minScore": 0}],
+              "unminified-javascript": ["error", {"minScore": 0}],
+              "unused-css-rules": ["error", {"minScore": 0}],
+              "uses-text-compression": ["error", {"minScore": 0}],
+              "uses-http2": ["error", {"minScore": 0}],
+              "errors-in-console": ["error", {"minScore": 0}]
+            }
+          },
+          {
+            "matchingUrlPattern": "http://[^/]+/terms$",
+            "assertions": {
+              "offscreen-images": ["error", {"minScore": 1}],
+              "uses-webp-images": ["error", {"minScore": 1}],
+              "render-blocking-resources": ["error", {"minScore": 0}],
+              "unminified-css": ["error", {"minScore": 0}],
+              "unminified-javascript": ["error", {"minScore": 0}],
+              "unused-css-rules": ["error", {"minScore": 0}],
+              "uses-text-compression": ["error", {"minScore": 0}],
+              "uses-http2": ["error", {"minScore": 0}],
+              "errors-in-console": ["error", {"minScore": 0}]
+            }
+          },
+          {
+            "matchingUrlPattern": "http://[^/]+/thanks$",
+            "assertions": {
+              "offscreen-images": ["error", {"minScore": 0}],
+              "uses-webp-images": ["error", {"minScore": 1}],
+              "render-blocking-resources": ["error", {"minScore": 0}],
+              "unminified-css": ["error", {"minScore": 0}],
+              "unminified-javascript": ["error", {"minScore": 0}],
+              "unused-css-rules": ["error", {"minScore": 0}],
+              "uses-text-compression": ["error", {"minScore": 0}],
+              "uses-http2": ["error", {"minScore": 0}],
+              "errors-in-console": ["error", {"minScore": 0}]
+            }
+          }
+        ]
+      },
+      "upload": {
+        "target": "temporary-public-storage"
+      }
+    }
```

Comment:
Two spaces to left, please try to preserve that each level is shifted by 2 spaces.